drivers/npmx/shell/ldsw.c

```diff
@@ -162,13 +187,38 @@ static int cmd_ldsw_gpio_get(const struct shell *shell, size_t argc, char **argv
 		return 0;
 	}
 
-	int8_t gpio_index =
-		(gpio_config.gpio == NPMX_LDSW_GPIO_NC) ? -1 : ((int8_t)gpio_config.gpio - 1);
+	if (param == LDSW_GPIO_PARAM_INDEX) {
+		int8_t gpio_index = (gpio_config.gpio == NPMX_LDSW_GPIO_NC) ?
+					    -1 :
+					    ((int8_t)gpio_config.gpio - 1);
+
+		shell_print(shell, "Value: %d", gpio_index);
+	} else if (param == LDSW_GPIO_PARAM_POLARITY) {
+		shell_print(shell, "Value: %d", gpio_config.inverted);
+	}
 
-	shell_print(shell, "Value: %d %d.", gpio_index, gpio_config.inverted);
 	return 0;
 }
 
+static int cmd_ldsw_gpio_index_set(const struct shell *shell, size_t argc, char **argv)
+{
+	return ldsw_gpio_set(shell, argc, argv, LDSW_GPIO_PARAM_INDEX);
+}
+
+static int cmd_ldsw_gpio_polarity_set(const struct shell *shell, size_t argc, char **argv)
+{
+	return ldsw_gpio_set(shell, argc, argv, LDSW_GPIO_PARAM_POLARITY);
+}
+
+static int cmd_ldsw_gpio_index_get(const struct shell *shell, size_t argc, char **argv)
+{
+	return ldsw_gpio_get(shell, argc, argv, LDSW_GPIO_PARAM_INDEX);
+}
+
+static int cmd_ldsw_gpio_polarity_get(const struct shell *shell, size_t argc, char **argv)
+{
+	return ldsw_gpio_get(shell, argc, argv, LDSW_GPIO_PARAM_POLARITY);
+}
```

Comment:
Wrong ordering, should be:
- `cmd_ldsw_gpio_index_set`
- `cmd_ldsw_gpio_index_get`
- `cmd_ldsw_gpio_polarity_set`
- `cmd_ldsw_gpio_polarity_get`
To match the ordering of commands' creation.
Also in Buck commit